catalyst/dl/callbacks/metrics/mrr.py

```diff
@@ -0,0 +1,40 @@
+from typing import List
+
+from catalyst.core import MetricCallback
+from catalyst.utils import metrics
+
+
+class MRRCallback(MetricCallback):
```

Comment:
could you please add this callback to the docs?
https://github.com/catalyst-team/catalyst/blob/master/docs/api/dl.rst#metrics